### ISSUE TITLE
Possible to run in non-interactive mode?

### ISSUE THREAD

**Issue by trbjo** [velum/lguf-brightness#4]
I would like to be able to bind hotkeys to increment or decrement the brightness.

This would allow me to just press XF86MonBrightnessDown or XF86MonBrightnessUp directly on the keyboard instead of having to fire up a terminal and running the program interactively each time.

So a sort of command mode. E.g. `/usr/bin/lguf_brightness --increment`.

Is this possible?

**Comment by phil9909** [comment]
@trobjo I'm looking for a similar feature. Have you solved this?
Otherwise I'm willing to implement this.

**Comment by jrdoane** [comment]
I'm not sure if this would be helpful, but I forked the repo and made some changes because I needed to support multiple displays and I wanted to be able to specify a percentage of max brightness without the interactive UI. C++ isn't really my strong suit, but I needed to find a way from keeping my 5K Ultrafines from burning holes in my retinas from the brightness, so I ripped out a bunch of the stepping code and simplified it.

So basically if I want 50% brightness, it's just `sudo lguf_brightness all 50`.

https://github.com/jrdoane/lguf-brightness
